guide/lib/examples/password.rb

```diff
@@ -0,0 +1,31 @@
+module Examples
+  module Password
+    def default_password
+      <<~PASSWORD
+        = f.govuk_password_field :password, label: { text: "Enter your password" }
```

Comment:
To be consistent with the example used in the Design System documentation.

```suggestion
        = f.govuk_password_field :password, label: { text: "Password" }
```